guides/source/association_basics.md

```diff
@@ -2178,8 +2178,7 @@ The `collection.where` method finds objects within the collection based on the c
 
 ##### `collection.exists?(...)`
 
-The `collection.exists?` method checks whether an object meeting the supplied
-conditions exists in the collection. It uses the same syntax and options as
```

Comment:
We're losing the note about being able to supply a condition to `exists?` with the change here so the new documentation doesn't read better to me. Perhaps the following change is sufficient to clear up the doubts here? 

```suggestion
The `collection.exists?` method checks whether an object within the collection by querying the table. It uses the same syntax and options as
```